rats-tls/dist/deb/debian/rules

```diff
@@ -7,10 +7,12 @@ BUILD_ROOT := $(CURDIR)/debian/rats-tls
 override_dh_auto_clean:
 
 override_dh_strip:
-	dh_strip --exclude=sgx_stub_enclave.signed.so --exclude=rats-tls-server --exclude=rats-tls-client --exclude=librats_tls.so* --exclude=libtls_wrapper*.so* --exclude=libcrypto_wrapper*.so* --exclude=libattester*.so* --exclude=libverifier*.so*
+	dh_strip --exclude=rats-tls-server --exclude=rats-tls-client --exclude=librats_tls.so* --exclude=libtls_wrapper*.so* --exclude=libcrypto_wrapper*.so* --exclude=libattester*.so* --exclude=libverifier*.so*
 
 override_dh_auto_build:
-	make -C rats-tls SGX=1
+	cd rats-tls && cmake -H. -Bbuild
 
 override_dh_auto_install:
-	Rats_Tls_Root=$(BUILD_ROOT)/opt/rats-tls Rats_Tls_Bindir=$(BUILD_ROOT)/usr/share/rats-tls/samples make -C rats-tls install
+	cd rats-tls && make -C build DESTDIR=$(BUILD_ROOT) install
+
+override_dh_usrlocal:
```

Comment:
Please type command `dpkg -r rats_tls` to see if `/usr/share/rats-tls/` and ` /usr/local/include/rats-tls/`, `/usr/local/lib/rats-tls` are removed.
If not, maybe you should add `postrm` file in Debian. Please refer to the [link\(https://github.com/alibaba/inclavare-containers/blob/master/shim/dist/deb/debian/postrm) for more information.

Reply:
I've already tested dpkg -r rats-tls, they are all removed